pkg/issuer/acme/dns/rfc2136/rfc2136.go

```diff
@@ -156,47 +137,31 @@ func NewDNSProviderCredentials(nameserver, tsigAlgorithm, tsigKeyName, tsigSecre
 	}
 	d.tsigAlgorithm = tsigAlgorithm
 
-	d.dns01Nameservers = dns01Nameservers
-
 	klog.V(5).Infof("DNSProvider nameserver:       %s\n", d.nameserver)
 	klog.V(5).Infof("            tsigAlgorithm:    %s\n", d.tsigAlgorithm)
 	klog.V(5).Infof("            tsigKeyName:      %s\n", d.tsigKeyName)
-	if klog.V(5) {
```

Comment:
Here I've just removed use of `klog.V(5)` as a bool, as in a future version of klog this syntax will no longer work (this ended out being included as I was testing some things locally)